requirements/edx/github.in

```diff
@@ -57,7 +57,7 @@ git+https://github.com/edx/openedx-chem.git@ff4e3a03d3c7610e47a9af08eb648d8aabe2
 
 # Third-party:
 -e git+https://github.com/edx/django-wiki.git@v0.0.23#egg=django-wiki
-git+https://github.com/edx/MongoDBProxy.git@25b99097615bda06bd7cdfe5669ed80dc2a7fed0#egg=MongoDBProxy==0.1.0
+git+https://github.com/edx/MongoDBProxy.git@c4e48e6c124c321b390d44de919fd9f404841781#egg=MongoDBProxy==0.2.0
```

Comment:
Is this version number going to cause us problems if upstream eventually does a new release we want to switch to?  The canonical solution to this is to use something like "==0.1.0+edx.2" (see the django-celery URL elsewhere in this file).  Then we can just increment the version number in our suffix as we make new releases.  For more details: https://www.python.org/dev/peps/pep-0440/#local-version-identifiers

Reply:
Will do.